test/e2e/webdriver/driver.js

```diff
@@ -749,16 +749,20 @@ class Driver {
    *
    * @param {string} [page] - its optional parameter to specify the page you want to navigate.
    * Defaults to home if no other page is specified.
+   * @param {boolean} [waitForControllers] - optional parameter to specify whether to wait for the controllers to be loaded.
+   * Defaults to true.
    * @returns {Promise} promise resolves when the page has finished loading
    * @throws {Error} Will throw an error if the navigation fails or the page does not load within the timeout period.
    */
-  async navigate(page = PAGES.HOME) {
+  async navigate(page = PAGES.HOME, waitForControllers = true) {
```

Comment:
Thoughts on using an options bag here instead of a boolean parameter? e.g. `navigate(page, { waitForControllers: true })`

That would make it easier to tell what the second parameter is. `navigate(page, false)` is a bit mysterious. I had to go reference this definition to understand it.

Later we could move the page into the options parameter as well, though maybe that's best done separately due to how disruptive it would be

Reply:
woah that is a great suggestion!! I was also a bit unsure about how I set it up as it was not 100% clear and had to add the comment to clarify. This makes much more sense!
I agree on your suggestion, I think that for now we can keep the page outside, and in a dedicate PR we can move the page in the options